src/main/resources/application.conf

```diff
@@ -0,0 +1,16 @@
+akka {
+    loglevel = WARNING
+}
+
+database {
+    connectionPool = disabled
```

Comment:
Why disabled? For what purpose connection pools are used?

Reply:
I had some problems during configuration with connectionPools which I later fixed and forgot to change it to e.g HikariCP. Connection pooling are used for managing active connections and returning them instead of creating new ones which is time consuming.